src/Raven.Server/RavenServer.cs

```diff
@@ -829,13 +828,18 @@ private void SendErrorIfPossible(TcpConnectionOptions tcp, Exception e)
             return stream;
         }
 
-        private bool TryAuthorize(RavenConfiguration configuration, Stream stream, TcpConnectionHeaderMessage header, out string msg)
+        private bool TryAuthorize(RavenConfiguration configuration, TcpConnectionOptions tcp, TcpConnectionHeaderMessage header, out string msg)
```

Comment:
Here we're checking whether client comes from what's set up in `UnsecureAddressAllowed`. The check whether server url is within that range is in `SecurityConfiguration.Validate()`.